<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RutasWidget</class>
 <widget class="QWidget" name="RutasWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1220</width>
    <height>785</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Nueva Ruta</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="lblNombreRuta">
            <property name="font">
             <font>
              <family>Montserrat</family>
              <pointsize>16</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Nombre:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="leNombreRuta">
            <property name="font">
             <font>
              <family>Montserrat</family>
              <pointsize>12</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lblOrigen">
            <property name="font">
             <font>
              <family>Montserrat</family>
              <pointsize>16</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Origen:  </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="leOrigen">
            <property name="font">
             <font>
              <family>Montserrat</family>
              <pointsize>12</pointsize>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="lblConductor_2">
            <property name="font">
             <font>
              <family>Montserrat</family>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Nueva Parada: </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QPushButton" name="btnAgregarParada">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>60</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Montserrat</family>
                <pointsize>13</pointsize>
                <strikeout>false</strikeout>
                <fontweight>Black</fontweight>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: #3b82f6; color: white;</string>
              </property>
              <property name="text">
               <string>+</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnEliminarParada">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>60</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Montserrat</family>
                <pointsize>13</pointsize>
                <strikeout>false</strikeout>
                <fontweight>Black</fontweight>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 52, 25); color: white;</string>
              </property>
              <property name="text">
               <string>-</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLineEdit" name="leNuevaParada">
          <property name="font">
           <font>
            <family>Montserrat</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="placeholderText">
           <string>Escribe una direcci√≥n y pulsa + </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listParadas">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>70</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnGuardarRuta">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>60</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Montserrat</family>
            <pointsize>13</pointsize>
            <strikeout>false</strikeout>
            <fontweight>Black</fontweight>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #3b82f6; color: white;</string>
          </property>
          <property name="text">
           <string>Guardar Ruta</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QWebEngineView" name="webMapRuta">
        <property name="minimumSize">
         <size>
          <width>600</width>
          <height>400</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>800</width>
          <height>600</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
